tests/e2e/integration/names.js

```diff
@@ -1,10 +1,11 @@
-const name = 'test.chain';
+const name = 'test1.chain';
+const resultName = 'test1\u200E.chain';
```

Comment:
I was going to say that it breaks text if I copy it from there, but actually, the text in TruncateMid is not selectable because of before/after.